src/main/java/com/st/utopia/cancellationservice/service/TicketService.java

```diff
@@ -0,0 +1,41 @@
+package com.st.utopia.cancellationservice.service;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.st.utopia.cancellationservice.dao.TicketDao;
+import com.st.utopia.cancellationservice.model.Ticket;
+
+@Service
+public class TicketService {
+	/**
+	 * The DAO for the "Ticket" table.
+	 */
+	@Autowired
+	private TicketDao ticketDao;
+	/**
+	 * Logger for handling errors in the Service layer.
+	 */
+	private static final Logger LOGGER = Logger.getLogger(TicketService.class.getName());
+	/**
+	 * Cancel Ticket function.
+	 */
+	public void cancelReservation(int flight, int row, char seat){
+		ticketDao.cancel(flight, row, seat);

```

Comment:
It hardly seems worth it to have a service-layer method for this when all of the input-validation logic is in the controller (and in the DAO, but it only logs failed invariants and preconditions instead of signalling them to its callers).